shopify_python/google_styleguide.py

```diff
@@ -63,6 +63,9 @@ class GoogleStyleGuideChecker(checkers.BaseChecker):
                   'finally-too-long',
                   "The larger the 'finally' body size, the more likely that an exception will be raised during "
                   "resource cleanup activities."),
+        'C6010': ('Multiple items imported from %(module)s in one import statement.',
```

Comment:
These don't commonly end in a period and are usually shorter, e.g.:
```
shopify_python/google_styleguide.py:192:0 I0011(locally-disabled) Locally disabling no-member (E1101)
```

If printed this would look like:
```
shopify_python/google_styleguide.py:192:0 C6010(multiple-import-items) Multiple items imported from os in one import statement. (C6010)
```

Maybe rephrase as:

```
Statement imports multiple items from '%(module)s'
```

or


```
Multiple items imported from '%(module)s'
```